.env.example

```diff
@@ -17,3 +17,6 @@ VITE_UNCHAINED_ARBITRUM_NOVA_HTTP_URL=https://dev-api.arbitrum-nova.shapeshift.c
 VITE_UNCHAINED_BASE_HTTP_URL=https://dev-api.base.shapeshift.com
 
 VITE_AGENTIC_SERVER_BASE_URL=http://localhost:8080
+
+VITE_FEATURE_ENABLE_SIDEBAR_LEFT=false
+VITE_FEATURE_ENABLE_SIDEBAR_RIGHT=false
```

Comment:
@reallybeard FYI, since we're not really doing anything with this for the time being and threads are now managed by assistant-ui, might as well